project/standard/templates/build.sh

```diff
@@ -11,6 +11,9 @@ npm run lint
 if [ $# -eq 0 ]
   then
     mvn clean install -Dmapstore2.version=$VERSION
-  else
-    mvn clean install -Dmapstore2.version=$1
+    elif [ $# -eq 1 ]
+        then
+            mvn clean install -Dmapstore2.version=$1
+        else
+            mvn clean install -Dmapstore2.version=$1 -P$2
```

Comment:
I tested the new pom.xml in a project and the printing is working with the profile added to the .war.
I 'm not sure if the indent here is correct of elif and else and apart of that I'm not sure we need the cases for the custom version. My proposal is to simplify with something like this:
```bash
if [ $# -eq 0 ]
  then
    mvn clean install -Dmapstore2.version=$VERSION
  else
    mvn clean install -Dmapstore2.version=$VERSION -P$1
fi
```